build.sc

```diff
@@ -143,6 +143,42 @@ object generator extends SwamModule with PublishModule {
 
 }
 
+
+
+object cli extends SwamModule{
+
+  def moduleDeps = Seq(text, core, runtime)
+
+  def publishVersion = swamVersion
+
+  def artifactName = "swam-cli"
+
+  def ivyDeps = Agg(
+    ivy"com.github.pureconfig::pureconfig-enumeratum:$pureconfigVersion",
+    ivy"com.github.scopt::scopt:3.7.1"
+  )
+
+  def pomSettings =
+    PomSettings(
+      description = "Swam code generator library",
+      organization = "org.gnieh",
+      url = swamUrl,
+      licenses = Seq(swamLicense),
+      versionControl = VersionControl.github("satabin", "swam"),
+      developers = Seq(swamDeveloper)
+    )
+
+  object test extends Tests with ScalafmtModule {
```

Comment:
This module doesn't have tests if I am not wrong. So you can just delete this object.